dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/util/MemoryInformation.java

```diff
@@ -419,13 +560,12 @@ private long applySwappiness(long cached) throws IOException, MemoryInformationE
 
     public static synchronized int getSwappiness() throws MemoryInformationException {
         try {
-            List<String> swappinessLines = FileSystem.readAllLines("/proc/sys/vm/swappiness");
-            if (swappinessLines.size() == 1) {
-                return Integer.parseInt(swappinessLines.get(0));
-            } else {
-                throw new MemoryInformationException(MessageFormat.format(BootstrapConstants.messages.getString("memory.information.unexpected"), "/proc/sys/vm/swappiness",
-                                                                          swappinessLines));
+            // Use cgroup swappiness if available
+            String fileToRead = "/proc/sys/vm/swappiness";
+            if (FileSystem.fileExists("/sys/fs/cgroup/memory/memory.swappiness")) {
+                fileToRead = "/proc/sys/vm/swappiness";
```

Comment:
Should this refer to the cgroup swappiness file instead?

Reply:
Great catch!